docs/FunctionalTestBuildScripts.md

```diff
@@ -70,6 +70,36 @@ A simple hello world program, compiled with `clang 14.0.0` that generates a .exe
 `clang-cl -o clangcl.14.pe.c.codeview.pdbpagesize_default.exe -fuse-ld=lld-link helloc.c -m32 -Z7 -MTd /link /CETCOMPAT /guard:cf /PDB:clangcl.14.pe.c.codeview.pdbpagesize_default.exe.pdb`  
 `clang-cl -o clangcl.14.pe.c.codeview.pdbpagesize_8192_pdbmissing.exe -fuse-ld=lld-link helloc.c -m32 -Z7 -MTd /link /CETCOMPAT /guard:cf /PdbPageSize:8192 /PDB:clangcl.14.pe.c.codeview.pdbpagesize_8192_pdbmissing.exe.pdb (then delete the pdb)`
 
+## gcc.example2.fnostackprotector
```

Comment:
>example2 [](http://example.com/codeflow?start=7&length=8)

nit: use name with exmaple1 since this is only 1 file, too keep with existing pattern

Reply:
pleas wait my PR.